lib/process-services/karma.conf.js

```diff
@@ -15,7 +16,7 @@ module.exports = function (config) {
                 included: true,
                 watched: false
             },
-            { pattern: 'node_modules/chart.js/dist/Chart.js', included: true, watched: false },
+            { pattern: 'node_modules/chart.js/dist/Chart.min.js', included: true, watched: false },
```

Comment:
I don't see .min.js file in the node modules, is that a new version?

Reply:
I followed the suggestion 'cause it was making it fail the whole task, but thinking again now, do we actually need this? i've run the unit test without and they work fine